aries_vcx_core/Cargo.toml

```diff
@@ -39,6 +40,7 @@ lazy_static = "1.4.0"
 derive_builder = "0.12.0"
 uuid = { version = "1.3.0", default-features = false, features = ["v4"] }
 tokio = { version = "1.20" }
+indy-vdr-proxy-client = { git = "https://github.com/mirgee/indy-vdr.git", rev = "fab0535", optional = true }
```

Comment:
Very cool 👍 but I've noticed you perhaps by mistake created PR against your own repo, rather against hyperledger origin https://github.com/mirgee/indy-vdr/pull/1

Reply:
This is on purpose, I first want to make the maintainers are actually interested in the client before making a PR.

Reply:
In the end decided to open a PR against the Hyperledger repo. But IMO merging it should not be in any way a precondition for merging this PR.

Reply:
agreed

Reply:
I agree, but it would be good to make a comment above this dependency; stating the changes you made and also that it is future TODO to monitor your indy-vdr PR and revert to using the main branch if/when your indy-vdr PR is merged